### DISCUSSION TITLE
I'm getting a network error on login this morning

### DISCUSSION THREAD

**Discussion by andresud** [supabase/supabase#1546] [Questions] [answered]
Request URL: https://xmmdsfpofzgxvouqkmjl.supabase.co/auth/v1/token?grant_type=password
Network request failed
Failed to load resource: net::ERR_CONNECTION_TIMED_OUT

No changes made on code, was working until I turn off my computer yesterday.

Please advise.

Thank you.

**Comment by steve-chavez** [comment, marked as answer]
Hey @andresud,

I've just sent you an email. Let me know if it's all working now.

**Reply by steve-chavez** [reply]
All good now. As discussed on email with Andre.

**Reply by andresud** [reply]
Thank you so much... it's on again!

**Reply by emschwartz** [reply]
I'm seeing a similar issues for some of my users (`Network request failed`). @steve-chavez is there something I can / need to do on my end to resolve this?

**Reply by steve-chavez** [reply]
@emschwartz What's your project ref?

**Reply by emschwartz** [reply]
@steve-chavez sent it to the email listed in your github profile

**Reply by eliasljubec** [reply]
@steve-chavez Hi Steve, I am facing a similar problem, could you share your solution?

**Reply by steve-chavez** [reply]
@eljubec Please send an email to support@supabase.com with more details.
